services/self-service/src/main/java/com/epam/dlab/backendapi/service/impl/BillingServiceImpl.java

```diff
@@ -142,7 +142,7 @@ public BillingReport getExploratoryBillingData(String project, String endpoint,
         resourceNames.add(exploratoryName);
         List<BillingReportLine> billingData = billingDAO.findBillingData(project, endpoint, resourceNames)
                 .stream()
-                .peek(bd -> bd.setCost(BigDecimal.valueOf(bd.getCost()).setScale(2, BigDecimal.ROUND_HALF_UP).doubleValue()))
+                .peek(bd -> bd.setCost(BigDecimal.valueOf(bd.getCost()).doubleValue()))
                 .collect(Collectors.toList());
```

Comment:
Do we need to start stream() then?

Reply:
Later billingData object is used for cost calculation and getting currency value

Reply:
the stream has been removed